src/material/core/option/option.scss

```diff
@@ -2,7 +2,10 @@
 @use '@angular/cdk';
 @use '@material/list/evolution-mixins' as mdc-list-mixins;
 @use '@material/list/evolution-variables' as mdc-list-variables;
+@use '@material/typography/typography' as mdc-typography;
 
+@use '../tokens/m2/mat/option' as tokens-mat-option;
```

Comment:
I see that there currently are no unthemeable tokens, but we probably want to put a call to emit them in case any get added in the future

Reply:
It may make more sense to do a pass over the whole library once we're done adding tokens, because we have a lot of places where we don't emit any tokens because the token map is empty.